main.js

```diff
@@ -1,64 +1,69 @@
-const bookList = document.querySelector('.books-list');
-const form = document.querySelector('form');
+const bookLists = document.querySelector('.books-list');
 const bookTitle = document.querySelector('.book-title');
 const bookAuthor = document.querySelector('.book-author');
+const addBookBtn = document.querySelector('.add-book-btn');
 
-// array object
-const books = [];
+// retrieve data
+const RetrieveBooks = JSON.parse(localStorage.getItem('books'));
 
-// function to store books localStorage
-function storeBooks() {
-  localStorage.setItem('books', JSON.stringify(books));
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  // add book
+  addBook(title, author) {
+    this.new = 'new';
+    const newBook = { title, author };
+    if (localStorage.getItem('books')) {
+      let availableBooks = JSON.parse(localStorage.getItem('books'));
+      availableBooks = [newBook, ...availableBooks];
+      localStorage.setItem('books', JSON.stringify(availableBooks));
+    } else {
+      localStorage.setItem('books', JSON.stringify([newBook]));
+    }
+  }
+
+  // remove Book
+  removeBook(id) {
+    this.new = 'new';
+    const bookList = RetrieveBooks.filter(
+      (book) => book.title !== RetrieveBooks[id].title,
+    );
+    localStorage.setItem('books', JSON.stringify(bookList));
+  }
```

Comment:
- [ ] Duplicated books(same title) are all removed when one of them is clicked. When a user clicks on the remove button, only that book should be removed. Kindly fix that issue.